src/components/table/table-row/table-row.style.js

```diff
@@ -0,0 +1,166 @@
+    ${applyRowStyling}
+  }
+`;
+
+function applyRowStyling(props) {
+  const { theme } = props;
+  return css`
+    .custom-drag-layer {
+      & {
+        background-color: #002E41;
+        cursor: grabbing;
+        cursor: -moz-grabbing;
+        cursor: -webkit-grabbing;
+        display: block;
+      }
+
+      ${StyledTableCell} {
+        background-color: #002E41;
+      }
+
+      .configurable-item-row__content-wrapper {
+        visibility: visible;
+      }
+    }
+
+    &:first-child ${StyledTableHeader} {
+      &:first-child {
+        border-radius: 0px 0 0 0;
+      }
+
+      &:last-child {
+        border-radius: 0 0px 0 0;
+      }
+    }
+
+    ${isClassic(theme) ? applyClassicRowStyling() : applyModernRowStyling(theme)}
+    ${selectableRowStyling}
+    ${selectedRowStyling}
+    ${highlightRowStyling}
+    ${dragRowStyling}
+  `;
+}
+
+function selectableRowStyling({ theme }) {
+  return css`
+    ${StyledTableCell}:first-child,
+    ${StyledTableHeader}:first-child {
+      &:not(.carbon-table-cell--select) {
+        ${isClassic(theme) ? 'padding-left: 15px' : ''};
+      }
+    }
+
+    .carbon-table-cell--select:first-child,
+    carbon-table-cell--select:first-child {
+        text-align: center;
+        width: 18px;
+      
+        .carbon-checkbox {
+          height: 15px;
+          padding-top: 0;
+        }
+      }
+    }
+  `;
+}
+
+function selectedRowStyling({ theme }) {
+  return css`
+    .carbon-table-row--selected,
+    .carbon-table-row--selected:nth-child(odd),
+    .carbon-table-row--selected:hover {
+      ${StyledTableCell} {
+        ${isClassic(theme) ? applyClassicSelectedStyling(theme) : applyModernSelectedStyling(theme)}
+        position: relative;
+    
+        &:before {
+          content: "";
+          height: 1px;
+          left: 0;
+          position: absolute;
+          top: -1px;
+          width: 100%;
+        }
+      }
+    }
+  `;
+}
+
+function highlightRowStyling({ theme }) {
+  const { table } = theme;
+  return css`
+    && .carbon-table-row--clickable {
+      cursor: pointer;
+    }
+
+    && .carbon-table-row--highlighted {
+      ${StyledTableCell} {
+        ${isClassic(theme) ? applyClassicHighlightStyling() : applyModernSelectedStyling(theme)}
+        position: relative;
+  
+        &:before {
+          content: "";
+          height: 1px;
+          left: 0;
+          position: absolute;
+          top: -1px;
+          width: 100%;
+        }
+      }
+  
+      &:hover {
+        ${StyledTableCell} {
+          background-color: ${isClassic(theme) ? '#D0E3FA' : table.selected};
+        }
+      }
+    }
+  `;
+}
+
+function dragRowStyling() {
+  return css`
+    .carbon-table-row--dragging {
+      user-select: none;
+    }
+
+    .carbon-table-row--dragged {
+      ${StyledTableCell} {
+        visibility: hidden;
+      }
+
+      + .carbon-table-row--dragging {
+        ${StyledTableCell} {
+          border-top: 1px solid #000A0E;
```

Comment:
Shouldn't we avoid using hardcoded colors outside of  the classic theme?

Reply:
Already addressed in previous reviews

Reply:
@benwilsonsage this is an example of some of the colours that are used in both classic and modern tables yet have no DLS definition